tests/components/steam_online/test_config_flow.py

```diff
@@ -0,0 +1,222 @@
+"""Test Steam config flow."""
+import steam
+
+from homeassistant import data_entry_flow
+from homeassistant.components.steam_online.const import CONF_ACCOUNTS, DOMAIN
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_REAUTH, SOURCE_USER
+from homeassistant.const import CONF_API_KEY, CONF_SOURCE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import (
+    ACCOUNT_1,
+    ACCOUNT_2,
+    ACCOUNT_NAME_1,
+    CONF_DATA,
+    CONF_IMPORT_DATA,
+    CONF_IMPORT_OPTIONS,
+    CONF_OPTIONS,
+    CONF_OPTIONS_2,
+    create_entry,
+    patch_interface,
+    patch_user_interface_null,
+)
+
+
+async def test_flow_user(hass: HomeAssistant) -> None:
+    """Test user initialized flow."""
+    with patch_interface():
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": SOURCE_USER},
+        )
+        result = await hass.config_entries.flow.async_configure(
```

Comment:
Please patch the integration setup function on all create entry results to avoid setting up the whole integration. See the scaffold script test template for example.